Pod/Classes/URBNArrayDataSourceAdapter.h

```diff
@@ -34,6 +34,13 @@
 - (void)appendItems:(NSArray *)newItems inSection:(NSInteger)section;
 
 /**
+ * Appends a new section with the new items. If not sectioned, this does nothing.
```

Comment:
Does nothing? Why? Most of these methods manipulate the main items array if not sectioned. Would this not be desirable?